Mathlib/MeasureTheory/Function/L1Space.lean

```diff
@@ -939,6 +939,43 @@ theorem coe_toNNReal_ae_eq {f : α → ℝ≥0∞} (hf : ∀ᵐ x ∂μ, f x < 
   simp only [hx.ne, Ne, not_false_iff, coe_toNNReal]
 #align measure_theory.coe_to_nnreal_ae_eq MeasureTheory.coe_toNNReal_ae_eq
 
+section count
+
+variable [MeasurableSingletonClass α] {f : α → β}
+
+/-- A function has finite integral for the counting measure iff its norm is summable. -/
+lemma hasFiniteIntegral_count_iff :
+    HasFiniteIntegral f Measure.count ↔ Summable (‖f ·‖) := by
```

Comment:
Could you provide this lemma (and the ones below) for `Measure.average` as well?

Reply:
Actually it looks like I was misunderstanding in our zulip discussion: `MeasureTheory.average` is not a measure in its own right, but an operator on functions defined as `(μ univ).toReal⁻¹ • ∫ x, f x ∂μ`. So I think there isn't an analogue.

Reply:
Oh I see 😢